internal/config/config.go

```diff
@@ -723,6 +734,34 @@ func (c AzureSEVSNP) GetMeasurements() measurements.M {
 	return c.Measurements
 }
 
+// SetMeasurements updates a config's measurements using the given measurements.
+func (c *AzureSEVSNP) SetMeasurements(m measurements.M) {
+	c.Measurements = m
+}
+
+// NewerThan returns true if the config is newer than the given config.
+func (c AzureSEVSNP) NewerThan(old AttestationCfg) (bool, error) {
+	otherCfg, ok := old.(*AzureSEVSNP)
+	if !ok {
+		return false, fmt.Errorf("cannot compare %T with %T", c, old)
+	}
+
+	if c.FirmwareSignerConfig.EnforcementPolicy == idkeydigest.MAAFallback &&
+		otherCfg.FirmwareSignerConfig.EnforcementPolicy != idkeydigest.MAAFallback {
+		return false, errors.New("cannot update enforcement policy to 'maaFallback'")
+	}
```

Comment:
```suggestion
```
I guess this isn't needed anymore since you decided to always create an MAA provider?